tests/unit/TechRecordsService.unitTest.js

```diff
@@ -8,21 +8,57 @@ const path = require('path')
 
 describe('getTechRecordsList', () => {
   var techRecordsDAOMock = new TechRecordsDAOMock()
+  const techRecordsMock = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../resources/technical-records.json'), 'utf8'))
 
   context('when db call returns data', () => {
     it('should return a populated response', () => {
-      const techRecordsMock = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../resources/technical-records.json'), 'utf8'))
+      var techRecordsService = new TechRecordsService(techRecordsDAOMock)
       techRecordsDAOMock.techRecordsMock = [techRecordsMock[0]]
       techRecordsDAOMock.numberOfrecords = 1
       techRecordsDAOMock.numberOfScannedRecords = 1
-      var techRecordsService = new TechRecordsService(techRecordsDAOMock)
       return techRecordsService.getTechRecordsList('1B7GG36N12S678410', 'current')
         .then((returnedRecords) => {
           expect(returnedRecords).to.not.equal(undefined)
           expect(returnedRecords).to.not.equal({})
           expect(returnedRecords).to.equal(techRecordsDAOMock.techRecordsMock[0])
         })
     })
+
+    context('and the statusCode by which we query is provisional_over_current', () => {
+      context('and the result is a techRecord with one provisional entry and one current one', () => {
+        it('should return the provisional entry', () => {
+          var techRecordsService = new TechRecordsService(techRecordsDAOMock)
```

Comment:
please use 'let' as previously related .. also please initialise in before each like the previous example we discussed , else parallel test will be affected